src/vorta/i18n/ts/vorta.de.ts

```diff
@@ -1144,47 +1139,47 @@
     </message>
     <message>
-        <location filename="../../borg/create.py" line="136"/>
-        <source>Starting backup..</source>
-        <translation>Starte Datensicherung..</translation>
-    </message>
-    <message>
-        <location filename="../../borg/umount.py" line="26"/>
+        <location filename="../../borg/umount.py" line="27"/>
         <source>No active Borg mounts found.</source>
         <translation>Keine aktiven Borg-Einhängepunkte gefunden.</translation>
     </message>
+    <message>
+        <location filename="../../borg/create.py" line="136"/>
+        <source>Starting backup...</source>
+        <translation>Starte Datensicherung...</translation>
+    </message>
 </context>
 <context>
     <name>settings</name>
     <message>
-        <location filename="../../models.py" line="204"/>
-        <source>Use light system tray icon (applies after restart, useful for dark themes).</source>
-        <translation>Helles System-Tray-Icon benutzen (wird nach Neustart angewandt, nützlich für dunkle Themes).</translation>
+        <location filename="../../models.py" line="195"/>
+        <source>Use light system tray icon (applies after restart, useful for dark themes)</source>
+        <translation>Helles System-Tray-Icon benutzen (wird nach Neustart angewandt, nützlich für dunkle Themes)</translation>
     </message>
     <message>
-        <location filename="../../models.py" line="209"/>
-        <source>Display notifications when background tasks fail.</source>
-        <translation>Benachrichtigungen anzeigen, wenn Hintergrund-Aufgaben fehlschlagen.</translation>
+        <location filename="../../models.py" line="200"/>
+        <source>Display notifications when background tasks fail</source>
+        <translation>Benachrichtigungen anzeigen, wenn Hintergrund-Aufgaben fehlschlagen</translation>
     </message>
     <message>
-        <location filename="../../models.py" line="214"/>
-        <source>Also notify about successful background tasks.</source>
-        <translation>Auch über erfolgreiche Hintergrund-Aufgaben benachrichtigen.</translation>
+        <location filename="../../models.py" line="205"/>
+        <source>Also notify about successful background tasks</source>
+        <translation>Auch über erfolgreiche Hintergrund-Aufgaben benachrichtigen</translation>
     </message>
     <message>
-        <location filename="../../models.py" line="222"/>
-        <source>Add Vorta to Login Items in Preferences &gt; Users and Groups &gt; Login Items.</source>
-        <translation>Füge Vorta zu Login Items in Preferences &gt; Users and Groups &gt; Login Items hinzu.</translation>
+        <location filename="../../models.py" line="213"/>
+        <source>Add Vorta to Login Items in Preferences &gt; Users and Groups &gt; Login Items</source>
+        <translation>Starte Vorta automatisch bei der Anmeldung (via Einstellungen &gt; Benutzer und Gruppen &gt; Autostart)</translation>
```

Comment:
maybe translate the german back to english and use that as original string?

Reply:
Good idea. I also dropped the last part.